shared/chat/conversation/messages/wrapper/long-pressable/index.native.tsx

```diff
@@ -1,18 +1,38 @@
 import * as React from 'react'
 import * as Kb from '../../../../../common-adapters/mobile.native'
 import * as Styles from '../../../../../styles'
+import Swipeable from 'react-native-gesture-handler/Swipeable'
+
+const _renderLeftActions = () => {
+  return (
+    <Kb.Box2 direction="horizontal">
+      <Kb.Icon type="iconfont-reply" style={styles.replyIcon} />
+    </Kb.Box2>
+  )
+}
 
 // See '.js.flow' for explanation
-const LongPressable = (props: {children: React.ElementType}) => {
+const LongPressable = (props: {children: React.ElementType; onSwipeRight: () => void}) => {
   const {children, ...rest} = props
+  const swipeable = React.useRef()
+  const _onLeftOpen = () => {
```

Comment:
we usually use the `_` prefix to indicate a module/class private thing. Since this is a closure variable i don't think its necessary. Tangentially TS supports `private` class member annotations so we can use that going forward in the class case.